### PR TITLE
Update for 1.18.2

### PR DESCRIPTION
**Pull request by theCALiC0**
Noticed Campanion doesn't work on 1.18.2, so I updated Dossier and Campanion. More testing may be necessary, but I believe that the tag changes should be all that is needed here. That said, since this is a breaking change as it uses different classes in the tag bits, so I recommend (and have done in the build.gradle for testing) a version bump to 0.6.0.

### CODE REVIEW

**Comment by theCALiC0** [comment]
PR for Campanion: https://github.com/TerraformersMC/Campanion/pull/163

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (jhbuchanan45) [inline .vscode/launch.json:1]: Looks like these files in `.vscode/` slipped through the cracks, use `git rm -r --cached <folder>` to remove them and commit again. I've had this issue before, I think it has to do with git only ignoring adding new files, not ones that have already been added to a commit.

**Review by jhbuchanan45 — CHANGES_REQUESTED** [review]
@Jade-TheCat everything looks fine here, assuming it works with Campanion and you can get the VSCode files removed, then I'll go annoy someone else who can actually merge it. 

I'm not sure if I'd call these changes breaking, it looks like just a change in the minecraft mappings from Tag to TagKey, but we'll see what NeusFear thinks (or whoever does the merge)

**Review by NeusFear — APPROVED** [review]
Agreed once you can remove the vscode files this should be mergable

**Comment by theCALiC0** [comment]
Closing since a 1.18.2 update has already gone through! Thank y'all!

**Pull request closed without merging**